set.mm

```diff
@@ -54084,20 +54084,35 @@ the restriction (of the relation) to the singleton containing this
 
   ${
     $d A x y $.
-    $( The restricted identity expressed with the class builder.  (Contributed
-       by FL, 25-Apr-2012.) $)
-    opabresid $p |- { <. x , y >. | ( x e. A /\ y = x ) } = ( _I |` A ) $=
-      ( weq copab cres cv wcel cid resopab equcom opabbii eqtr4i reseq1i eqtr3i
-      wa df-id ) BADZABEZCFAGCHRPABEICFRABCJSICSABDZABEIRTABBAKLABQMNO $.
+    $( The restricted identity relation expressed as an ordered-pair class
+       abstraction.  (Contributed by FL, 25-Apr-2012.) $)
+    opabresid $p |- ( _I |` A ) = { <. x , y >. | ( x e. A /\ y = x ) } $=
+      ( cid cres weq copab cv wcel df-id equcom opabbii eqtri reseq1i resopab
+      wa ) DCEBAFZABGZCEAHCIQPABGDRCDABFZABGRABJSQABABKLMNQABCOM $.
   $}
 
   ${
     $d A x y $.
-    $( The restricted identity expressed with the maps-to notation.
+    $( The restricted identity relation expressed in maps-to notation.
        (Contributed by FL, 25-Apr-2012.) $)
-    mptresid $p |- ( x e. A |-> x ) = ( _I |` A ) $=
-      ( vy cv cmpt wcel weq wa copab cid cres df-mpt opabresid eqtri ) ABADZEOB
-      FCAGHACIJBKACBOLACBMN $.
+    mptresid $p |- ( _I |` A ) = ( x e. A |-> x ) $=
+      ( vy cid cres cv wcel weq wa copab cmpt opabresid df-mpt eqtr4i ) DBEAFZB
+      GCAHIACJABOKACBLACBOMN $.
+  $}
+
+  ${
+    $d A x y $.
+    $( Obsolete version of ~ opabresid as of 26-Dec-2023.  (Contributed by FL,
+       25-Apr-2012.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    opabresidOLD $p |- { <. x , y >. | ( x e. A /\ y = x ) } = ( _I |` A ) $=
+      ( cid cres cv wcel weq wa copab opabresid eqcomi ) DCEAFCGBAHIABJABCKL $.
+
+    $( Obsolete version of ~ mptresid as of 26-Dec-2023.  (Contributed by FL,
+       25-Apr-2012.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    mptresidOLD $p |- ( x e. A |-> x ) = ( _I |` A ) $=
```

Comment:
mptresidOLD depends on the new version introduced later?

This is the true OLD version (some $ characters vanish here due to some automatic replacements made by the online editor):

>   ${
>    $d A x y $.
>    $( The restricted identity expressed with the maps-to notation.
>       (Contributed by FL, 25-Apr-2012.) $)
>    mptresid $p |- ( x e. A |-> x ) = ( _I |` A ) $=
>      ( vy cv cmpt wcel weq wa copab cid cres df-mpt opabresid eqtri ) ABADZEOB
>      FCAGHACIJBKACBOLACBMN $.>
>  $}

Reply:
Thanks for noticing. I restored both old proofs.